src/App.js

```diff
@@ -59,7 +76,9 @@ export default function App() {
   function navigateHome() {
     history.push('/')
   }
-
+  function navigateForm() {
+    history.push('/form')
+  }
```

Comment:
shift to component
probably the same with the `navigateForm` function

also I think this is more a `link` than a `button` with a function